tests/network/hotplug_sriov.go

```diff
@@ -35,7 +35,7 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
 
```

Comment:
nit: remove newline

Reply:
The newline separates production from e2e tests packages.